package.json

```diff
@@ -6,13 +6,23 @@
   "repository": "https://github.com/suda/toolbar",
   "license": "MIT",
   "keywords": [
-    "toolbar"
+    "toolbar",
+    "tool-bar"
   ],
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">=0.188.0"
   },
   "dependencies": {
-    "atom-space-pen-views": "^2.0.3",
+    "grim": "^1.2.2",
+    "space-pen": "^5.1.1",
     "underscore-plus": "^1.6.6"
+  },
+  "providedServices": {
+    "tool-bar": {
```

Comment:
hmmm, I don't think apm is ready for renamed packages. I updated the packages via Atom available update (thus not via git) and I got two tool-bars and two times the main tool-bar (see screenshot). I also got folders two of both packages (see screenshot too). And lost my settings (ain't really bad, as it is only two settings).

![tool-bar](https://cloud.githubusercontent.com/assets/55841/7442043/add9a39c-f101-11e4-82d1-d270a04f6c78.png)

I think it's an acceptable change, but now you know.